src/app.module.ts

```diff
@@ -2,10 +2,13 @@ import { Module } from '@nestjs/common';
 import { CouchProxyController } from './couch-proxy/couch-proxy.controller';
 import { HttpModule } from '@nestjs/axios';
 import { DocumentFilterService } from './document-filter/document-filter.service';
+import { SessionModule } from './session/session.module';
 
 @Module({
-  imports: [HttpModule],
+  imports: [HttpModule, SessionModule],
   controllers: [CouchProxyController],
   providers: [DocumentFilterService],
 })
 export class AppModule {}
+
+export const COUCH_ENDPOINT = 'https://dev.aam-digital.com/db';
```

Comment:
Let's use environment variables on the server-side services (probably using dotenv https://nodejs.dev/learn/how-to-read-environment-variables-from-nodejs)

Reply:
#4